internal/publish/model/exposure_model.go

```diff
@@ -17,17 +17,78 @@ package model
 
 import (
 	"context"
+	"encoding/base64"
 	"fmt"
+	"math"
 	"sort"
 	"strings"
 	"time"
 
 	"github.com/google/exposure-notifications-server/internal/logging"
+	"github.com/google/exposure-notifications-server/internal/verification"
 	"github.com/google/exposure-notifications-server/pkg/base64util"
 
 	verifyapi "github.com/google/exposure-notifications-server/pkg/api/v1alpha1"
 )
 
+// Exposure represents the record as stored in the database
+// TODO(mikehelmick) - refactor this so that there is a public
+// Exposure struct that doesn't have public fields and an
+// internal struct that does. Separate out the database model
+// from direct access.
+// Mark records as writable/nowritable - is exposure key encrypted.
+type Exposure struct {
+	ExposureKey      []byte
+	TransmissionRisk int
+	AppPackageName   string
+	Regions          []string
+	IntervalNumber   int32
```

Comment:
Why int32? Why not uint?

Reply:
this has always been int32 - and aligns w/ the proto type in the export file.